packages/cdk8s-cli/README.md

```diff
@@ -32,24 +34,148 @@ $ cd my-fun-little-project
 $ cdk8s init typescript-app
 ```
 
-## import - Import API objects as constructs
+## import
+
+### Selecting import language
+
+You can specify the language of your imports with the `--language` flag.
+
+`cdk8s` CLI
+
+```code
+cdk8s import --language typescript
+```
+
+`cdk8s.yaml` Config
+
+```yaml
+language: typescript
```

Comment:
Language is probably going to be used for other things as well so I wouldn’t put it under the imports section.